package.json

```diff
@@ -27,7 +27,8 @@
     "level-js": "~4.0.1",
```

Comment:
Can `leveljs` and `leveldown` be deleted?

Reply:
Yes, they can